bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -3947,6 +3963,10 @@ public void removeTourModifySelectionListener(final ITourModifyListener listener
       _tourModifyListener.remove(listener);
    }
 
+   public void removeTourPauseSelectionListener(final ITourPauseSelectionListener listener) {
```

Comment:
@wolfgang-ch I didn't see the equivalent `removeTourMarkerSelectionListener` used anywhere is it not ?

Reply:
Currently there is no need to call `removeTourMarkerSelectionListener` because this listener is used only in a dialog and when this is closed, the tour chart is also discarded

![image](https://user-images.githubusercontent.com/1283445/133726821-dbc3bb06-7638-492f-97d4-8bd09c9c2bd7.png)

Reply:
Should it be removed from the code then ??

Reply:
I think for listener we should keep it, these add/remove methods are created manually and not automatically like set/get methods

Reply:
Ok, I kept those